src/presenters/Whip_WPMessagePresenter.php

```diff
@@ -19,20 +19,33 @@ public function __construct( Whip_Message $message ) {
 	/**
 	 * Registers hooks to WordPress. This is a separate function so you can
 	 * control when the hooks are registered.
+	 *
+	 * @param Whip_MessageDismisser $dismisser Dismisser object.
 	 */
-	public function register_hooks() {
+	public function register_hooks( Whip_MessageDismisser $dismisser ) {
```

Comment:
The `$dismisser` should be injected through the constructor as a 2nd argument and stored as a property.